plugin/mlsdbonlinelocator.cpp

```diff
@@ -45,8 +46,22 @@ MlsdbOnlineLocator::MlsdbOnlineLocator(QObject *parent)
     , m_modemManager(new QOfonoExtModemManager(this))
     , m_simManager(0)
     , m_networkManager(new NetworkManager(this))
+    , m_fallbacksLacf(true)
+    , m_fallbacksIpf(true)
     , m_currentReply(0)
 {
+    QString MLSConfigFile = QStringLiteral("/etc/gps_xtra.ini");
+    QSettings settings(MLSConfigFile, QSettings::IniFormat);
+    if(settings.contains("MLS/FALLBACKS_LACF")) {
```

Comment:
Coding style nit picking. There should be a space in between of "if(" => "if ("

Reply:
Ok. Thank you